internal/wasmdebug/dwarf.go

```diff
@@ -17,7 +17,9 @@ type DWARFLines struct {
 	// linesPerEntry maps dwarf.Offset for dwarf.Entry to the list of lines contained by the entry.
 	// The value is sorted in the increasing order by the address.
 	linesPerEntry map[dwarf.Offset][]line
-	mux           sync.Mutex
+	// Cache for Language(). nil when Language() has never been called.
+	lang *DwarfLang
```

Comment:
Any reason not to use `DwarfLangUnkonwn` instead of a pointer to indicate that the value is uninitialized?

Reply:
Ah sorry I meant to add a comment about this.

It is to distinguish between "Language() has not been called" and "DWARF info doesn't say anything about language". Could have also used a bit flag or an extra boolean variable, but that seemed the most straightforward that way.